### ISSUE TITLE
Serving model when group size != 1. 

### ISSUE THREAD

**Issue by nalin-mathur** [tensorflow/ranking#109]
I have successfully trained various different LeToR models using the example training script provided in the repo. I then used the models that are saved in 'outdir/export/latest_exporter/' to make predictions on new data. 

However, when training the model with a group_size != 1, the same files are not saved in the outdir. The script also mentions in a comment that in order to serve the model, group size should be 1. 
From the code, I can understand that the group_size is used as the output of the neural network. Thus a group size of x shall have number of logits layers equal to x. Hence group_size is essentially the dimensionality of the output space. 

Why does the group_size need to be 1 in order to serve the model? And is there any way to work around this limitation such that a model trained with group_size != 1 can be used for further predictions?

**Comment by ramakumar1729** [comment]
Closing this issue for now.
